product_docs/docs/pem/9/monitoring_performance/notifications.mdx

```diff
@@ -197,6 +197,9 @@ You must configure the PEM server to send the notifications to an SNMP trap/noti
        createUser -e 0x50454D5F534E4D505F4147454E540A pem_snmp_user MD5 pem_auth_pass DES pem_priv_pass
        authUser log pem_snmp_user
     ```
+!!!note
+SNMP V3 can not work with an agent running as non-root.
```

Comment:
@edbshubham  What do you think about rephrasing a bit to this? 

```suggestion
The agent must run as root to allow sending notifications with SNMP V3.
```

Reply:
looks great..Thanks